components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2829,7 +2829,9 @@ void RewardsServiceImpl::OnPublisherListNormalizedSaved(
 
   ContentSiteList site_list;
   for (auto& publisher : *list) {
-    site_list.push_back(PublisherInfoToContentSite(publisher));
+    if (publisher.percent >= 1) {
```

Comment:
I don't think this is the right place to do this. This is UI logic and should be filtered by the UI, not by RewardsServiceImpl

Reply:
also there should be a test for this

Reply:
this was added here as other (sort is here as well) that is then sent to UI. Left comment here how I want to solve it https://github.com/brave/brave-core/pull/2432/files#r283678712

Reply:
The sort is also here which is UI, but I think even short-term it would be better to move both of those to the UI class OnPublisherListNormalized observer method. I think we're all on the same page now that we should be doing the calculations at query time? If not let's discuss in slack with @tmancey as well